copier/main.py

```diff
@@ -745,6 +751,105 @@ def run_update(self) -> None:
             self.template.migration_tasks("after", self.subproject.template)
         )
 
+    def _apply_update_inline_conflict_markers(self):
```

Comment:
This function is the only meaningful new code in the PR. It's a new/different implementation of the existing `Worker.apply_update()` function.

Reply:
Thanks, may you please add docstrings here too?

Reply:
Done!